src/backend/lib/db_helpers/case_manager.ts

```diff
@@ -40,7 +40,7 @@ export async function createPara(
         role: "staff",
       })
       .returningAll()
-      .executeTakeFirst();
+      .executeTakeFirstOrThrow();
```

Comment:
We currently don't handle any cases where we test for it returning 'undefined', therefore I think having it throw and forcing us to deal with the error is a better option